tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/op/NameScope.java

```diff
@@ -50,6 +53,45 @@ NameScope withName(String name) {
     return new NameScope(opPrefix, name, ids);
   }
 
+  private static final Pattern NAME_PATTERN = Pattern.compile("(.+)_(\\d+)", Pattern.DOTALL);
+
+  /**
+   * "Import" used names from a graph.  Useful when adding to a loaded graph.
+   */
+  NameScope withUsedFrom(ExecutionEnvironment env) {
+
+    if (env instanceof Graph) {
```

Comment:
Use `env.isGraph()` instead?

Reply:
I'm casting it, so I didn't want to rely on `isGraph` always mapping directly to `instanceof Graph`.  I can though if you don't think it will change.